dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/ds3Panel/CreateService.java

```diff
@@ -28,11 +26,14 @@
 
 public final class CreateService {
```

Comment:
Not in this pull request, but we need to turn this into a real service where any of the internal dependencies get injected, and more importantly, where all the ui logic is removed.